apps/web/lang/en.json

```diff
@@ -622,6 +622,16 @@
     "vat": {
       "incl": "incl.",
       "plus": "plus"
+    },
+    "upload": {
+      "orderPropertyTypeFile":"Order Property - Type File",
+      "uploading": "uploading",
+      "uploaded": "uploaded",
+      "orderProperty":"Order Property",
+      "dragAndDropFileHereOr":"Drag and drop file here or",
```

Comment:
Can we work with a placeholder here?
e.g. 'Drag and drop file here or {uploadFile}'
maybe there is a language were the sentence structure is different?

Reply:
Interesting ideea but for this particular case if you want to respect the mockups the answer is no,
because you need separate css like color to font, then it needs to be diferent element.

https://www.figma.com/file/q93sg5sfWaa4BSbTayJLEM/plentyShop-PWA-Mockups?type=design&node-id=553-1785&mode=design&t=KrV7L4GZ9JKPugCw-0

Reply:
Is possible with something like this:
`<i18n-t keypath="yourTextWithPlaceholderInIt">
     <template #yourPlaceholder>
         here your normal html for your separate color and so on
      </template>
</i18n-t>`